src/helpers/JSON-performance-report-parser.helper.spec.js

```diff
@@ -0,0 +1,19 @@
+import JSONPerformanceReportParser from './JSON-performance-report-parser.helper';
+import { expect } from 'chai';
+import fs from 'fs';
+
+const parser = new JSONPerformanceReportParser();
+
+describe('JSON performance report parser', () => {
+  it('returns found objects with TTFB and URL values', (fileName = 'performance-report.har') => {
+    const parsedReport = JSON.parse(fs.readFileSync(`src/tests/reports/performance/${fileName}`, 'utf8'));
+
+    expect(parser.mapRequests(parsedReport, fileName)).to.deep.include({ ttfb: 0, url: 'http://localhost:8080/' });
```

Comment:
Again, you're testing method that should be a private one. Ask yourself a question, what method are you going to use as a gate to parser `mapRequests` or `parse`. I'm 100% percent sure it will be `parse` so you should test only `parse` method. 

You should pass only a file name here and test for expeced result, not parse the report here and expect that it maps properly. Right now the `parse` method lacks tests and this is the one that is going to be used.